### ISSUE TITLE
"Invited" field (for uninvited publications) not mapping correctly from Elements to VTW

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#290]
Hi, @amandafrench , there's an issue with the mapping of the "c-invited" (boolean) field from Elements to VTW. I recall earlier that you mentioned that you didn't want the value "false (Invited?)" to appear for conference publications that were not invited. I tried to adjust for this in the crosswalk, but my adjustment did not work. This mapping IS working correctly for invited publications, but that was never an issue.

I'll put in a support ticket on Elements' site, but I'd recommend that we move forward with the Irish Setter deployment and fix this issue during our next sprint. If Symplectic can't find a way to adjust the mapping so that no information is exported if the "Invited?" box is not checked, I recommend removing this field altogether from the mapping since we're not collecting this information for other conference materials in VTW. **EDIT** On the other hand, we could simply map this field to "dc.description.provenance" so that only we would see it. Thanks!

**Comment by keithgee** [comment]
@mello99 Please try again. It looks like an older version of the crosswalk was installed.

**Comment by mello99** [comment]
How can I see which version of the crosswalk has been installed? I'd like to make sure that it's the correct one. Is the crosswalk that's installed currently on the "vt_5_x_dev" branch?

**Comment by mello99** [comment]
It still didn't work :(. I put in a support ticket on Symplectic's website; in addition, I posted a question about this issue on the forums.

**Comment by keithgee** [comment]
Yes, that's the one that's installed. I compared the digest of the file actually on vtechworks.dev with the digest of the one that is on the vt_5_x_dev branch. They match.

You have to have shell access to vtechworks.dev to do this:

```
vtkeithg@vtechworks:~$ cd /var/tmp
vtkeithg@vtechworks:/var/tmp$ md5sum /dspace/config/crosswalks/symplectic_xwalks_dspace_default.xsl
4de468bc40b42a40c3f9c987ac0c5034  /dspace/config/crosswalks/symplectic_xwalks_dspace_default.xsl
vtkeithg@vtechworks:/var/tmp$ wget https://raw.githubusercontent.com/VTUL/vtechworks/vt_5_x_dev/dspace/config/crosswalks/symplectic_xwalks_dspace_default.xsl
--2016-06-13 01:20:08--  https://raw.githubusercontent.com/VTUL/vtechworks/vt_5_x_dev/dspace/config/crosswalks/symplectic_xwalks_dspace_default.xsl
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 199.27.76.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|199.27.76.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 13048 (13K) [text/plain]
Saving to: ‘symplectic_xwalks_dspace_default.xsl’

100%[======================================>] 13,048      --.-K/s   in 0.02s   

2016-06-13 01:20:08 (825 KB/s) - ‘symplectic_xwalks_dspace_default.xsl’ saved [13048/13048]

vtkeithg@vtechworks:/var/tmp$ md5sum symplectic_xwalks_dspace_default.xsl 
4de468bc40b42a40c3f9c987ac0c5034  symplectic_xwalks_dspace_default.xsl
```

**Comment by mello99** [comment]
Closing, will reopen if necessary

**Comment by amandafrench** [comment]
Fixed by #310
